apps/ledger-live-desktop/tests/specs/accounts/account.spec.ts

```diff
@@ -9,7 +9,7 @@ import { AccountsPage } from "../../models/AccountsPage";
 
 test.use({ userdata: "skip-onboarding" });
 
-const currencies = ["BTC", "LTC", "ETH", "ATOM", "XTZ", "XRP", "Tron"];
+const currencies = ["BTC", "LTC", "ETH", "ATOM", "XTZ", "XRP", "Tron", "optimism", "kava evm"];
```

Comment:
are `optimism` and `kava evm` both ruled by the same family? Is there specific behavior difference of the **mocks** between these two? otherwise, I think we should only keep once, (same as we may consider removing LTC) because there is no benefit to test the same thing (it's a bit a fake test since we don't assess the coin implementation, but more the UI, and if UI don't differ we don't need to cover it) VS there is a cost on running these (it takes time)

Reply:
No, there are no specific behaviour difference between these two. I did these because of the "BTC" and "LTC" in bitcoin family. Then I'll remove one of them.